src/proxyClient.js

```diff
@@ -97,7 +97,14 @@ var frameId = 0;
 
 // Worker
 
-var worker = new Worker('{{{ filename }}}.js');
+var filename = '{{{ filename }}}.js';
+var workerURL = filename;
+var fileBytes = tryParseAsDataURI(filename);
+if (fileBytes) {
+  workerURL = URL.createObjectURL(new Blob([fileBytes], {type: 'application/javascript'}));
+}
+
+var worker = new Worker(workerURL);
```

Comment:
Cool, well, as with the question of `MEM_INIT_METHOD=2` deprecation, I'll leave this decision to you and @kripken before changing anything.